src/SearchField/SearchField.doc.js

```diff
@@ -23,14 +23,24 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'onBlur',
+        type: '(event: SyntheticEvent<{ target: HTMLInputElement }>) => void',
+      },
       {
         name: 'onChange',
-        type: '({ value: string }) => void',
+        type: `({
+          value: string,
+          event: SyntheticEvent<{ target: HTMLInputElement }>
```

Comment:
I don't think you need to add `<{ target: HTMLInputElement }>` here - just `<HTMLInputElement>` will match the code.

Reply:
Also, the key `event` should be `syntheticEvent`.